src/main/java/com/appium/capabilities/Capabilities.java

```diff
@@ -105,7 +112,7 @@ private void processJSONObject(JSONObject objectToUpdate,
                                    String keyStr,
                                    Object keyvalue) {
         currentPath.append(keyStr);
-        String getFromEnv = System.getenv(currentPath.toString());
+        String getFromEnv = Variable.getOverriddenStringValue(currentPath.toString());
```

Comment:
IMO, Class Name `Variable` is not a good selection.

Reply:
Any suggestion for a better name?

Reply:
Also, didn't understand why the CI is failing. Can you point me to where and why it is complaining?

Reply:
@anandbagmar 

>>INFO [main] (HostMachineDeviceManager.java:338)- Chrome not found on device - '4618862C-1074-4F9C-804C-7E2AEE964366'
java.util.NoSuchElementException: No value present
    at java.util.OptionalInt.getAsInt(OptionalInt.java:118)

This error is fixed in master. Can you please update your branch with master

Reply:
@saikrishna321 - I updated the latest ATD changes in my fork. Don't think it triggered the tests again though.

Reply:
@saikrishna321 , @SrinivasanTarget  - I have merged ATD master in the fork. Also renamed Variable class to OverriddenVariable. The tests are still failing in CI as there are no devices available for some of the pipeline jobs. Please take a look at that.